packages/node/test/integration/cant-take-action.spec.ts

```diff
@@ -30,7 +30,7 @@ describe("Node method follows spec - fails with improper action taken", () => {
         }
       };
       await createChannel(nodeA, nodeB);
-      const appInstanceId = await installApp(nodeA, nodeB, Apps.TicTacToe);
+      const [appInstanceId] = await installApp(nodeA, nodeB, Apps.TicTacToe);
```

Comment:
Why does this method now return an array with one element in it?

Reply:
The `installApp` function creates the proposal parameters that the proposal & install get called with, so if the caller cares about what those parameters were, it looks at the second element of this returned list.

Some callers care about this to test whether the installed app has the same parameters as the proposed parameters.